frontend/src/lib/api/book.js

```diff
@@ -0,0 +1,35 @@
+import client from './client';
+
+// Fetch list of books
+export const getList = () => {
```

Comment:
If the execution output returns in one line, we can write the code as below. There is no need for curly bracket here.
``` suggestion
export const getList = () => client.get('/books');
```